README.md

```diff
@@ -1,114 +1,112 @@
-# Welcome to the QuestCraft GitHub!
-### **[Support us on Patreon!](https://patreon.com/QuestCraftXR)**
-
-# QuestCraft
+ # Welcome to the QuestCraft GitHub!
 ![QC](/QCSimple3.jpg)
 
 QuestCraft, utilizing Vivecraft and Pojlib, is a standalone port of Minecraft: Java Edition to Oculus Quest Headsets. With a team of four main developers and 8 third-party developers, over the past year, we have not only gotten the bestselling game, Minecraft: Java Edition running on Android, but we have also worked with third-party developers to bring it into Virtual Reality on the Oculus Quest line of standalone HMDs.
 
+### **[Support us on Patreon!](https://patreon.com/QuestCraftXR)**
+
 # Features
 - ### **Roomscale Minecraft VR!** 
-- ### **Quest 2 Native!\***
+- ### **Quest 1/2/Pro/3 Native!\***
```

Comment:
quest 1 doesnt work as qc is rn

Reply:
Judge said it will work though caveats do exist. Remember: I'm trying to future proof this README.